fn/develop/configs.md

```diff
@@ -1,22 +1,115 @@
-# Function Config Vars
+# Use of environment variables
 
-There are three ways to get configuration variables into a function, all of which will be available as environment variables.
-These are ordered in order of preference, the later ones overriding the previous ones.
+If you want to use configuration variables into a function and make them available as environment variables, then you can set them using the Fn tool or the func.yaml file.
+ 
+By using the Fn tool, you can set configuration variables on the application or configuration level.
 
-## (1) Application level configuration
+# Set environment variables
 
+The following three examples shows you how to set the configuration variables on different levels using the `connection-app` application. These are listed in order of preference, the later ones overrides the previous ones:
```

Comment:
Suggested change: 

The following three examples show you how to set the configuration variables on different levels using the `connection-app` sample application. The configuration examples are listed in order of precedent, the later examples override the previous ones. For example, adding a function key/value pair overrides an application key/value pair with the same name.

Reply:
Done